docs/assets/search-data.js

```diff
@@ -36,15 +36,20 @@
 
   {{- $pages := where .Site.Pages "Kind" "in" (slice "page" "section") -}}
   {{- $pages = where $pages "Params.booksearchexclude" "!=" true -}}
-  {{- $pages = where $pages "Content" "not in" (slice nil "") -}}
```

Comment:
@MartijnVisser I could not work out how to get this filter fixed. I tried a bunch of things, including inverting the logic from `not in` to `in`, and simplifying to `(slice nil)`/`(slice "")`.  Could not get any results. I have refactored to a non-ideal loop with filter using a counter to generate the index. It works..... but it is not nice :D 

Any idea?